Base/res/fortunes.json

```diff
@@ -114,5 +114,80 @@
     "author": "BenW",
     "url": "https://discord.com/channels/830522505605283862/859531354219872266/889222244218327130",
     "utc_time": 1632077460
+  },
+  {
+    "quote": "You reached the end of the internet, turn around",
+    "author": "Hendiadyoin",
+    "url": "https://discord.com/channels/830522505605283862/830525031720943627/876969458432884766",
+    "utc_time": 1629156212,
+    "context": "An interesting Browser bug. https://discord.com/channels/830522505605283862/830525031720943627/876966237077389352"
+  },
+  {
+    "quote": "Widgets all inherit from Object, which has the methods for dealing with children",
+    "author": "AtkinsSJ",
+    "url": "https://discord.com/channels/830522505605283862/830529037251641374/882936925739171881",
+    "utc_time": 1630578967
+  },
+  {
+    "quote": "slaps roof of PR queue\nThis baby can hold so many e̴̾̓l̷͑͑d̸̉̚r̵̅͂ï̷̀t̸͗̉c̵̾͝h̶̋͗ ̸̲̚h̸͒́ȱ̸r̶̛̄ȓ̶͌o̶͐͘r̴̲͝s̴̐͆",
+    "author": "Andrew Kaster",
+    "url": "https://discord.com/channels/830522505605283862/830525235803586570/922976701032562738",
+    "utc_time": 1640125193
+  },
+  {
+    "quote": "Do you like bugs? yeah, me neither, so I fixed one:",
+    "author": "linusg",
+    "url": "https://discord.com/channels/830522505605283862/859531354219872266/922965066423341066",
+    "utc_time": 1640122419
+  },
+  {
+    "quote": "why is german",
+    "author": "sin-ack",
+    "url": "https://discord.com/channels/830522505605283862/830525235803586570/917781310766993408",
+    "utc_time": 1638886516
+  },
+  {
+    "quote": "\n> Lasciate ogni speranza, voi ch'entrate!\n> Abandon hope all ye who enter here\n~ Dante",
+    "author": "Hendiadyoin",
+    "url": "https://discord.com/channels/830522505605283862/830522505605283866/917781369084596284",
+    "utc_time": 1638886529,
+    "context": "Multithreaded IPC. https://discord.com/channels/830522505605283862/830522505605283866/917780454298484806"
+  },
+  {
+    "quote": "yakcoin is powered by proof of commits\neverytime you shave a yak you earn one yakcoin",
+    "author": "sin-ack",
+    "url": "https://discord.com/channels/830522505605283862/830525235803586570/849632952089837599",
+    "utc_time": 1622638681
+  },
+  {
+    "quote": "tootlips",
+    "author": "awesomekling",
+    "url": "https://discord.com/channels/830522505605283862/830739873119207426/895421091856666685",
+    "utc_time": 1633555424,
+    "context": "https://discord.com/channels/830522505605283862/830739873119207426/895421023833436160"
+  },
+  {
+    "quote": "and then the firebug nation attacked",
+    "author": "gggggg",
+    "url": "https://discord.com/channels/830522505605283862/830525031720943627/893862331312902166",
+    "utc_time": 1633183786
+  },
+  {
+    "quote": "there is no need to narrate all your random testing :)",
```

Comment:
not sure I'd include this one, it's oddly unexciting

Reply:
sure